src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
The `checkHealth` method should be a static method. This will allow it to be called on the `Animal` class itself, ensuring that the static `alive` array is updated correctly. Consider changing the method to `static checkHealth()` and updating its usage accordingly.